target/linux/generic/files/drivers/bcma/fallback-sprom.c

```diff
@@ -17,6 +17,9 @@
 #include <linux/of_net.h>
 #include <linux/of_platform.h>
 
+int __init bcma_fbs_register(void);
+int bcma_get_fallback_sprom(struct bcma_bus *dev, struct ssb_sprom *out);
+
```

Comment:
810-bcma-ssb-fallback-sprom.patch already adds prototypes for these functions in a header file.

Maybe this file is just missing an include for this header?

Also please always include the verbatim error messages in your commit messages.

Reply:
Moved to separate PR https://github.com/openwrt/openwrt/pull/17138 (commit c25e8f24fcfdbbd0482f0e3771400ef0b9addbf4)